languages/Transfer.groovy

```diff
@@ -67,6 +67,7 @@ buildList.each { buildFile ->
 		// evaluate the datasetmapping, which maps build files to targetDataset defintions
 		PropertyMappings dsMapping = new PropertyMappings("transfer_datasetMapping")
 		PropertyMappings dsOptionsMapping = new PropertyMappings("transfer_dsOptions")
+        PropertyMappings copyModeMapping = new PropertyMappings("transfer_copyMode")		
```

Comment:
I think we could/should simplify the implementation of the `copyMode` configuration by using the DBB file property syntax instead of the PropertyMappings API (both are pretty close) .

The implementation would be similar to the `deployType` configuration - see line 105. This allows to set the default in Transfer.properties (instead within the code) and use file property overrides it in file.properties.

@lauren-li @M-DLB

Reply:
I changed it to use a regular property. Indeed it's easier.
Can you review the change @dennis-behm ?